tests/integrations/raftstore/test_transport.rs

```diff
@@ -62,3 +65,17 @@ fn test_secure_connect() {
         must_get_equal(&cluster.get_engine(id), key, value);
     }
 }
+
+#[test]
+#[should_panic]
```

Comment:
This will make a panic stacktrace always appear in output. You can check status code instead.

Reply:
Fix